system-test/error-reporting.ts

```diff
@@ -551,7 +552,7 @@ describe('error-reporting', () => {
       errors.report(errOb, undefined, undefined, (err, response, body) => {
         assert.ifError(err);
         assert(is.object(response));
-        assert.deepEqual(body, {});
+        deepStrictEqual(body, {});
```

Comment:
I don't understand why this would be required here over `assert.deepStrictEqual`

Reply:
I think `assert.deepStrictEqual` would be ok, but I was using the new `deepStrictEqual` function throughout for consistency.